### ISSUE TITLE
Aries PWS URL Update

### ISSUE THREAD

**Issue by USAFPride** [sprinkler/rainmachine-developer-resources#28]
The URL is missing `pws_` for using the station ID. The urls (row 88 and 117

`"http://api.aerisapi.com/observations/summary/pws_" + stationName +  "`...

**Comment by ve7gel** [comment]
Yes, you need to enter that yourself, because not all station ID's have PWS as the prefix. Madis stations begin with MID_, and airports have no prefix.
